src/main/java/com/networknt/schema/OneOfValidator.java

```diff
@@ -308,8 +308,7 @@ private ValidationMessage getMultiSchemasValidErrorMsg(String at){
             msg = msg.concat(schemaValue);
         }
 
-        return ValidationMessage.of(getValidatorType().getValue(), ValidatorTypeCode.ONE_OF ,
-                at, String.format("but more than one schemas {%s} are valid ",msg));
+        return ValidationMessage.of(getValidatorType().getValue(), ValidatorTypeCode.ONE_OF, at, msg);

```

Comment:
creative solution 😄 have you verified that the compiled message is correctly compiled with all placeholder variables being substituted accordingly?

I'm asking because you noticed this bug due to your personal code returning an incompletely translated message. Then, I wouldn't have to check it myself.

Reply:
The message used to be before:

`requestBody.data darf nur für ein einziges Schema but more than one schemas {{"$ref":"#/components/schemas/tEnS"}{"$ref":"#/components/schemas/vEnS"}{"$ref":"#/components/schemas/vEnSAndTens"}} are valid gültig sein`

this means that variable msg contained the String `{"$ref":"#/components/schemas/tEnS"}{"$ref":"#/components/schemas/vEnS"}{"$ref":"#/components/schemas/vEnSAndTens"}`, and what we saw was a combination of the previous text for oneOf: "{0} darf nur für ein einziges Schema {1} gültig sein" and the result of String.format()...

So I changed the text for oneOf to "{0} darf nur für ein einziges Schema gültig sein, aber mehr als ein Schema ist gültig: {1}", and thus the error message now indeed in my scenario is the following:

`requestBody.data darf nur für ein einziges Schema gültig sein, aber mehr als ein Schema ist gültig: {"$ref":"#/components/schemas/tEnS"}{"$ref":"#/components/schemas/vEnS"}{"$ref":"#/components/schemas/vEnSAndTens"}`

(note that I left out the second pair of curly brackets, we can re-introduce them if you like)

If you would like me to create a test case that explicitly validates this, I am fine with this, but due to the fact that I have a hospital stay scheduled for next week, it might **only be at/around Easter** that I will be able to do so.

This would still be fine from my point of view, as I think the German translation for the validation messages is currently rather a "niche feature" of json-schema-validator 😄 ...

Reply:
Ah, and also to the best of my knowledge (checking search results for "oneOf" in the project) no other place in the code is using of the message resource with the "oneOf" key...